docs/source/x_ext.rst

```diff
@@ -338,6 +354,27 @@ An ``id`` ends being in-flight when one of the following scenarios apply:
 * the corresponding commit transaction killed the offloaded instruction and no corresponding memory request transaction and/or corresponding memory result transactions is in progress or still needs to be performed.
 * the corresponding result transaction has been performed.
 
+Superscalar and Multiple Harts
+------------------------------
+
+The interface can be used in scenarios where the |processor| is superscalar, i.e. it can issue more than one instruction per cycle.
+In such scenarios, the the |coprocessor| is usually required to also be able to accept more than one instruction per cycle.
+To support this, the entire interface is replicated according to the issue width. 
+It is possible to have a single issue |coprocessor| connected to a multi-issue |processor|, but then the latter is restricted to offloading at most one instruction per cycle.
+State within the |coprocessor| may not be duplicated, but must be consistent for all issue pipelines of the |coprocessor|.
```

Comment:
What does that 'consistency' mean, can you give an example?

If we are for example talking about an PFU including F registers, then state MUST be replicated, right?

Reply:
In the superscalar case, for F, the registers should not be duplicated. There is only one physical register file. You are still running one thread of instructions, just issuing two at a time.